src/components/CreateDataRequestForm.vue

```diff
@@ -19,23 +19,33 @@
     </el-form-item>
 
     <el-form-item label="Data request fee" prop="fee">
-      <el-input v-model.number="form.fee"></el-input>
+      <el-input v-model.number="form.fee">
+        <InputAppend slot="append" />
+      </el-input>
     </el-form-item>
 
     <el-form-item label="Reward fee" prop="rewardFee">
-      <el-input v-model.number="form.rewardFee"></el-input>
+      <el-input v-model.number="form.rewardFee">
+        <InputAppend slot="append" />
+      </el-input>
     </el-form-item>
 
     <el-form-item label="Commit fee" prop="commitFee">
-      <el-input v-model.number="form.commitFee"></el-input>
+      <el-input v-model.number="form.commitFee">
+        <InputAppend slot="append" />
+      </el-input>
     </el-form-item>
 
     <el-form-item label="Reveal fee" prop="revealFee">
-      <el-input v-model.number="form.revealFee"></el-input>
+      <el-input v-model.number="form.revealFee">
```

Comment:
We still need to solve the validation of numbers
<img width="500" alt="Captura de pantalla 2020-06-11 a las 11 11 39" src="https://user-images.githubusercontent.com/16032491/84367462-8f129e80-abd4-11ea-87ac-f5ad13cfa698.png">